src/data/faq.json

```diff
@@ -87,6 +87,117 @@
+                        ]
+                    },
+                    {
+                        "title": "How long is a rapid antigen test valid? What does the count-down mean?",
+                        "anchor": "rat_how_long_valid",
+                        "active": true,
+                        "textblock": [
+                            "A negative rapid test result is valid for 48 hours after submission and can be used as evidence, if required by law. When you tap on your negative test result on the main screen of the app, you will see a counter that shows you how long your test result has already been available. After 48 hours, your rapid test result will automatically be removed. You can then take another rapid test and register it in the app.",
+                            "There is no limited validity for positive rapid test results, so no counter will be displayed to you in this case. As soon as you receive a positive test result, immediately follow the instructions displayed in the app."
+                        ]
+                    },
+                    {
+                        "title": "What privileges derive from a negative rapid test result? Can I use my test result in the app as proof of admission to shops or visiting hair salons, leisure activities, etc.?",
+                        "anchor": "rat_privileges",
+                        "active": true,
+                        "textblock": [
+                            "A negative test result can serve as proof of the existence of a negative rapid test result for you, if specified by law. However, the recognition of test evidence may vary from state to state. You should therefore inform yourself about the respective criteria in your federal state."
+                        ]
+                    },
+                    {
+                        "title": "How do I act if my rapid test result is positive?",
+                        "anchor": "rat_what_if_positive_result",
+                        "active": true,
+                        "textblock": [
+                            "In case of a positive test result, you should share your test result and immediately take a PCR test to confirm or debunk the result. In case of a positive PCR test, you should share the test result again to alert contacts you may have met between the two tests. This is provided that the contacts also use the Corona Warn App and the risk calculation determines a high risk, based on duration and distance of contact. If the PCR test is negative, you will not have the option to withdraw your positive rapid test and warning."

```

Comment:
```suggestion
                            "In case of a positive test result, you should share your test result and immediately take a PCR test to confirm or debunk the result. In case of a positive PCR test, you should share the test result again to alert contacts you may have met between the two tests. This is provided that the contacts also use the Corona-Warn-App and the risk calculation determines a high risk, based on duration and distance of contact. If the PCR test is negative, you will not have the option to withdraw your positive rapid test and warning."

```